ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/DumpMetaData.java

```diff
@@ -38,7 +38,7 @@
 public class DumpMetaData {
   // wrapper class for reading and writing metadata about a dump
   // responsible for _dumpmetadata files
-  private static final String DUMP_METADATA = "_dumpmetadata";
+  public static final String DUMP_METADATA = "_dumpmetadata";
```

Comment:
why is this needed?

Reply:
The test for this patch (testDumpMetadataBackwardCompatibility) needs to probe on the dumpmetadata file.
The test works by doing a dump, then deleting newly introduced fields from the dumpmetadata file and then doing a load. This change was made to provide access to the dmd-file name to the test.